docs/source/share_kedro_viz.md

```diff
@@ -23,7 +23,26 @@ When prompted for a project name, you can enter anything, but we will assume `Sp
 
 When your project is ready, navigate to the root directory of the project.
 
-## Update and install the dependencies
+## Platform-Agnostic sharing with Kedro-viz 
+
+Starting from Kedro-Viz version 7.1.0, we introduced the `kedro viz build` command. When a developer runs this command from the command line interface (CLI), it will create a `build` folder within their Kedro project. This build folder contains the entire static Kedro-Viz app package, which users can manually publish to any hosting platform of their choice.
```

Comment:
[Nit] I am not sure of the tense but should it be - Starting from Kedro-Viz version 7.1.0, we **introduce** the `kedro viz build` command ?

Reply:
When a developer runs this command from the command line interface (CLI) -> When a **user** runs this command from the command line interface (CLI) ?

Reply:
This build folder contains the entire static Kedro-Viz app package which users can manually publish to any hosting platform of their choice. -> **The** build folder contains **a static Kedro-Viz app package, which can be used as a source to publish on any static website hosting platforms.**

Reply:
I'm resolving this as I edited this section so no longer applicable